crates/bevy_feathers/src/assets/shaders/color_plane.wgsl

```diff
@@ -0,0 +1,27 @@
+// This shader draws a checkerboard pattern
```

Comment:
This file might be more clearly titled as color_pane_checkerboard :)

Reply:
One thing you'll discover about my coding habits is that while I am really good at commenting my code, I'm not so good at spotting inaccuracies in the comments after cutting and pasting the code from somewhere else :)